### ISSUE TITLE
RFC: the default value of gitea_version

### ISSUE THREAD

**Issue by DO1JLR** [roles-ansible/ansible_role_gitea#46]
I open this issue, with the Request for Comments on the update behavior of gitea with this role.

**TL;DR**
- Keep the default value of ``gitea_version`` as it is?
- Change ``gitea_version`` to ``latest``?
- undefine ``gitea_version`` and force everyone to think about it?

----------------

Since the latest release [v2.4.0](https://github.com/roles-ansible/ansible_role_gitea/releases/tag/v2.4.0) of this Ansible role, the automatic update to the latest gitea release is now also possible.

But the default value of the ``gitea_version`` variable still contain an exact release of go-gitea. So with every update of gitea this release number must be adjusted. But unfortunately this does not always happen in a very timely manner. :worried: 

So now my thought would be to change the value of the ``gitea_version`` variable. :thinking:  But if it is just changed to ``'latest'``, then users of this role might not notice this and might install a gitea version at some point, which they would not have wanted to install without being asked.

On the other hand, the variable ``gitea_version`` could not be given a default value anymore. But explain to the users of this role in the README what values belong here *(latest or an exact release version)* and so you make no more decisions for the users of this role, which they may prefer to make themselves. :sunglasses: 

**What do you think about this?** Do you have an opinion that you would like to share?

**Comment by jogrie** [comment]
I personally like it, when roles work out of the box.
Also i played a lot with docker, and 'latest' is there the default unless you set a specific version.

So i would appreciate to set the version to 'latest' with a big disclaimer in the readme

**Comment by mar1ad** [comment]
After using it for a couple of months, I changed my mind.
I agree with @jogrie , 'latest' is indeed the more sensible default.
